src/Payum/Core/Bridge/Propel2/Model/PayumOrder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Payum\Core\Bridge\Propel2\Model;
+
+use Payum\Core\Bridge\Propel2\Model\Base\PayumOrder as BasePayumOrder;
+
+/**
+ * Skeleton subclass for representing a row from the 'payum_order' table.
+ *
+ *
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ *
+ */
+class PayumOrder extends BasePayumOrder
```

Comment:
No problem, but what about the sql table name ? Should I keep the payum_ prefix for ```payum_(order|token)``` to allow developpers to have a 1 database app or not ?
Here's a real example: In Thelia we already have an ```order``` table, and many of our users are on shared hostings, and only have 1 database, so we need to have a different name to allow our users to use payum with the propel 2 bridge.